aws-sdk-s3-transfer-manager/Cargo.toml

```diff
@@ -1,11 +1,16 @@
 [package]
 name = "aws-sdk-s3-transfer-manager"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
do we want the rust-version? https://github.com/awslabs/aws-sdk-rust/blob/main/sdk/s3/Cargo.toml#L10